GameLife.html

```diff
@@ -14,8 +14,8 @@ <h1 style="text-align:center; color: blue; ">
 	</h1>
 
 	<div style="text-align:center">
-		<button type="button" id="button" onclick="start()" style="height:30px; width:150px; color: blue;">Start!</button>
-		<button type="button" id="button" onclick="stop(stopInt)" style="height:30px; width:150px; color: red;">Stop!</button>
+		<button type="button" id="button1" onclick="myDisabled()" style="height:30px; width:150px; color: blue;">Start!</button>
```

Comment:
Give meaningful ids to buttons like buttonStart, buttonClose, not button1/2

Give meaningful names to functions like enable_start, enable_stop - use consistent name convention for all functions not some xxx_xxx and some XxxXxx.